log4j-docgen-maven-plugin/src/main/java/org/apache/logging/log4j/docgen/maven/SchemaGeneratorMojo.java

```diff
@@ -50,6 +50,9 @@ public class SchemaGeneratorMojo extends AbstractGeneratorMojo {
 
     @Override
     public void execute() throws MojoExecutionException {
+        if (skip) {
```

Comment:
Should we log this fact? E.g.:

```
getLog().info("Skipping schema generation.");
```

Reply:
Good idea! Implemented in 842e23fae62ba1ad217626fcd5c40d0983b15cdd.